src/test/java/uk/gov/ons/ctp/response/casesvc/service/impl/CaseServiceImplTest.java

```diff
@@ -331,34 +331,6 @@ public void testCreateCaseEventAgainstNonExistentCase() throws CTPException {
     assertNull(result);
   }
 
-  /**
-   * Tries to apply an actionable event against a case already inactionable. Should allow.
-   *
-   * @throws Exception if fabricateEvent does
-   */
-  @Test
-  public void testCreateActionableEventAgainstInactionableCase() throws Exception {
-    when(caseRepo.findOne(INACTIONABLE_HOUSEHOLD_CASE_FK))
```

Comment:
Why have we removed this entire unit test? It probably needs fixing, but isn't it testing a scenario which is still potentially valid?

Reply:
The functionality this was testing was removed. If you look in `CaseServiceImpl` we've removed the `createAdHocAction` function and event though the naming isn't the best, that was what was actually being tested. 
I checked against the data in prod and that function could never be hit, I think it was something left over from census

Reply:
ahhh... right. I guess this unit test would still hit a few of the things being tested, but I'll take your word for it that it's now redundant: I can't get my head around the naming and what exactly the test is trying to do.